src/app/services/author.service.ts

```diff
@@ -34,6 +34,6 @@ export class AuthorService {
           this.refreshRequired$.next();
         })
       )
-      .subscribe(data => { console.log('PUT request was successful. PUT: ', data); }, error => { console.log('Error', error); });
+      .subscribe();
```

Comment:
Prefer to always return the observable and let the user of the service call subscribe.

Reply:
Fixed this. Previously I had logged info and errors about requests, now the service user calls .subscribe on all service methods as needed.